baton.gemspec

```diff
@@ -22,17 +22,17 @@ Gem::Specification.new do |gem|
   gem.add_runtime_dependency "eventmachine", "~> 1.0.0"
   gem.add_runtime_dependency "em-http-request", "1.0.3"
   gem.add_runtime_dependency "bunny", "~> 0.8.0"
-  gem.add_runtime_dependency "chef", "~> 11.8"
+  gem.add_runtime_dependency "chef", "~> 12.2"
```

Comment:
This is the change needed to use a version of `ffi-yajl` gem that supports ruby 2.4